src/advanced_inv.cpp

```diff
@@ -1835,9 +1835,3 @@ void advanced_inv_area::set_container_position()
         veh = nullptr;
     }
 }
-
-void game::advanced_inv()
```

Comment:
This function is implemented in this file so no other file needs to include the advanced_inv.h header. That makes compiling a bit faster and reduces the dependencies.